conda/moose/meta.yaml

```diff
@@ -26,6 +26,7 @@ requirements:
   build:
     - {{ moose_libmesh }}
     - {{ moose_python }}
+    - {{ moose_wasp }}
     - moose-tools
```

Comment:
Yep, this is needed. In my current work with Kitware/Peacock these four lines become one: `{{ moose_dev }}`